### ISSUE TITLE
@step_from_factory no longer picks up arbitrary factory class names as step definitions

### ISSUE THREAD

**Issue by jscn** [koterpillar/aloe#110]
I upgraded from `0.0.34` to the most recent release and ran into the following behaviour: factories decorated with  `@step_from_factory` now fail to correctly infer the step text from the class name.

I have inherited some feature files with steps like:

```
Given I have nationwide services:
...
```

which has an associated factoryboy factory like:

```
@step_from_factory
class NationwideService(ServiceFactory):
    foo = bar
```

Prior to `0.1.15`, this was working fine. Now, however, I get the following error:

```
NoDefinitionFound: The step r"And I have nationwide services:" is not defined
```

I'm not sure if this change was intentional or not. It appears to have been introduced in 365a59b0cbf3348a1bf5ca5db3d9e0927d3dd3a4. By the looks of that commit, it fixed a bug which caused the decorator to _always_ infer the step name from the factory class name, not the model's name (https://github.com/aloetesting/aloe/commit/365a59b0cbf3348a1bf5ca5db3d9e0927d3dd3a4#diff-91332d13b9c03c246990815584252355R106).

Now, since there is a model called `Service`, that gets picked up and preferred over inferring the factory name from the class name of the factory (as per https://github.com/aloetesting/aloe/commit/365a59b0cbf3348a1bf5ca5db3d9e0927d3dd3a4#diff-91332d13b9c03c246990815584252355R108)

As I said, I'm not sure if this changes was intentional or not, but given that the author of those steps and factories is an author of this library, I'm assuming it was an unintentional side effect of that changeset.

**Comment by koterpillar** [comment]
As a workaround, you can do this: https://github.com/aloetesting/aloe/blob/master/tests/factoryboy_app/features/steps.py#L105

I'll check if I can get the class name to work too.

**Comment by koterpillar** [comment]
If the class you are creating is a Django model, its `verbose_name` might have been customized, and, as far as I know, there's no official way to find out. Aloe currently only looks at the factory name if it can't find the above (i.e. if you are creating something that's not a Django model).

I will be fixing the outdated documentation. As for the behavior, it's complicated if both the factory class name and the model's verbose name are customized.

**Comment by koterpillar** [comment]
Fixed in v0.1.6, see freshly minted changelog.
